### ISSUE TITLE
Ctrl-T should show the list of currently running tests

### ISSUE THREAD

**Issue by yurivict** [pytest-dev/pytest#12407]
#### What's the problem this feature will solve?
When some tests run for a very long time it is important to be able to know what are these tests. Currently it is difficult to find out.

#### Describe the solution you'd like
Pressing Ctrl-T should show the list of currently running tasks.

On FreeBSD we have a similar situation: the program that builds packages normally prints what packages were just finished or started, but when Ctrl-T is pressed it prints the list of currently running builds like this:

![image](https://github.com/pytest-dev/pytest/assets/271906/907cad9c-1d03-4812-9016-65d812c9a34b)

PyTest should do the same and on Ctrl-C should print the list of all currently running tests with their times, statuses, etc.

**Comment by RonnyPfannschmidt** [comment]
PTest itself is not parallel/concurrent 

However it's possibly a nice feature to have a hangup signal handler that triggers a hook which lets plugins inform the user of the current state 

Pytest itself could return the current test In non-verbose mode

xdist could return the active test per gateway

**Comment by nicoddemus** [comment]
As @RonnyPfannschmidt said, that feature would not make sense for pytest because it does not run tests in parallel (`pytest-xdist` and other plugins do, however).

I'm closing this for now, but another issue can be opened with a proposal for a new hook as outlined by @RonnyPfannschmidt. 👍

**Comment by yurivict** [comment]
> [...] but another issue can be opened with a proposal for a new hook [...]

I suggested a feature that would be potentially very useful.

How it is implemented and what hooks should be involved is an implementation detail.
I am **not** a PyTest developer and don't know what is the best way to implement this feature.
Developers should decide this, not me.

I don't know why this feature proposal was closed because this is a valid proposal for a needed, useful feature.
I don't know what hooks should be involved.

**Comment by nicoddemus** [comment]
I didn't mean for you to open a new issue, sorry @yurivict.

I just think that we probably should work on a proposal for the hook in a separate issue, otherwise this tends to get "lost" in the midst of unrelated discussion.

But in retrospect I should not have closed this, as you mention, indeed this describes the feature, which is independent on how it will end up being actually implemented.
